### PR TITLE
Bump up dependencies.

### PR DESCRIPTION
**Pull request by takayahilton**

### CODE REVIEW

File: project/AkkaDependency.scala [hunk 1]
```diff
@@ -14,7 +14,7 @@ object AkkaDependency {
   // else if akka version is "default", then the hard coded default will be used (jenkins doesn't allow empty values for config axis)
   // else if akka.version is anything else, then the given version will be used
 
-  val defaultAkkaVersion = "2.5.23"
+  val defaultAkkaVersion = "2.5.25"
```
Comment (jrudolph) [inline project/AkkaDependency.scala]: We usually bump the Akka version only if strictly needed.

**Comment by jrudolph** [comment]
OK TO TEST

File: project/AkkaDependency.scala (hunk 1 above)
Comment (raboof) [reply project/AkkaDependency.scala]:
```suggestion
  // Updated only when needed, https://github.com/akka/akka/issues/26985
  val defaultAkkaVersion = "2.5.25"
```

File: project/AkkaDependency.scala [hunk 2]
```diff
@@ -14,6 +14,7 @@ object AkkaDependency {
   // else if akka version is "default", then the hard coded default will be used (jenkins doesn't allow empty values for config axis)
   // else if akka.version is anything else, then the given version will be used
 
+  // Updated only when needed, https://github.com/akka/akka/issues/26985
```
Comment (jrudolph) [inline project/AkkaDependency.scala]: Great, thanks :+1:

**Comment by jrudolph** [comment]
Thanks, @takayahilton.

**Comment by laszlovandenhoek** [comment]
Jackson Databind 2.9.9 is still vulnerable to CVE-2019-14439. While we're upgrading, we should go for 2.9.9.3. I'll prepare a separate PR.

**Pull request merged**